customBoogie.patch

```diff
@@ -0,0 +1,385 @@
+diff --git a/Source/Dafny.sln b/Source/Dafny.sln
+index 50810fdc..da1a5def 100644
+--- a/Source/Dafny.sln
++++ b/Source/Dafny.sln
+@@ -26,6 +26,30 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DafnyTestGeneration", "Dafn
+ EndProject
+ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DafnyTestGeneration.Test", "DafnyTestGeneration.Test\DafnyTestGeneration.Test.csproj", "{896E7F24-FD59-4B34-A1BF-53C51DDBC9E9}"
+ EndProject
++Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Boogie", "Boogie", "{60332269-9C5D-465E-8582-01F9B738BD90}"
```

Comment:
I'm confused by this reference - is it referring to the `Boogie.sln` file?

Reply:
It creates a visual folder in which to put the Boogie projects, the parent-child relationships are somewhere lower in this file.

Reply:
Ah nice, I like it. It wasn't obvious from attempting to mentally parse this file. :)